rundk/README.md

```diff
@@ -0,0 +1,46 @@
+# rundk
+
+`rundk` is a tool to run a test command from the test image, by using it
+developers can reproduce the test flow as run in the CI environment.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/) must be installed
+- If you run this tool on MAC, `/tmp` directory must be added to Docker's File
+  Sharing settings to allow Docker accessing the temporary files created for
+  mount, see the [comment](https://github.com/docker/docker.github.io/issues/4709#issuecomment-639596451).
+
+## Installation
+
+`rundk` can be installed and upgraded by running:
+
+```shell
+go get knative.dev/test-infra/rundk
+```
+
+## Usage
+
+```shell
+Usage of rundk:
+  -test-image string
+        The image we use to run the test flow. (default "gcr.io/knative-tests/test-infra/prow-tests:stable")
+  -mounts string
```

Comment:
The likelihood the local directory has the same path as would be useful inside the image is unlikely, so I don't think this feature adds anything

Reply:
Users would probably want to mount other files when they run the tests? The flag is empty by default, just an extension in case users may want extra mounts.

Reply:
What files? The whole git directory gets mounted.
What good does mounting `/home/<myusername>/.kube/config` do, it's not put in a usable place?
You need to be able to define the external and internal paths separately to be useful.

Reply:
Changed the flag value to be in the format of ``source1:target1,source2:target2`.
Thanks.